File: .sdk/tm/dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinTecsMerchantServicesSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinTecsMerchantServicesSDK.test();
      equal(true, null != testsdk);
    });
  });
}

File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinTecsMerchantServices SDK exists test.

using Xunit;

using BluefinTecsMerchantServicesSdk;

namespace BluefinTecsMerchantServicesSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinTecsMerchantServicesSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
